server/views/pages/functionalSkills/index.njk

```diff
@@ -28,61 +28,114 @@
 
   {% include "../../partials/prisonerBanner.njk" %}
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <span class="govuk-caption-l">Learning and work progress</span>
-      <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+  {% if not allFunctionalSkills.problemRetrievingData %}
```

Comment:
On the Functional Skills page, I've now wrapped it in an if block (consistent with Education & Training; and Support Needs), where we show an error message if Curious was down and we could not get the Functional Skills data at all